catalog/core/catalog-core-standardframework/src/test/java/ddf/catalog/impl/operations/RemoteDeleteOperationsTest.java

```diff
@@ -0,0 +1,326 @@
+        frameworkProperties.setMimeTypeMapper(new MimeTypeMapperImpl(Collections.singletonList(
+                mimeTypeResolver)));
+        frameworkProperties.setMimeTypeToTransformerMapper(mimeTypeToTransformerMapper);
+        frameworkProperties.setValidationQueryFactory(new ValidationQueryFactory(new GeotoolsFilterAdapterImpl(),
+                new GeotoolsFilterBuilder()));
+
+        SourceOperations sourceOperations = new SourceOperations(frameworkProperties);
+        OperationsSecuritySupport opsSecurity = new OperationsSecuritySupport();
+        MetacardFactory metacardFactory = new MetacardFactory(mimeTypeToTransformerMapper);
+        OperationsMetacardSupport opsMetacard = new OperationsMetacardSupport(frameworkProperties,
+                metacardFactory);
+
+        QueryOperations queryOperations = new QueryOperations(frameworkProperties,
+                sourceOperations,
+                opsSecurity,
+                opsMetacard);
+
+        OperationsCatalogStoreSupport opsCatStore = new OperationsCatalogStoreSupport(
+                frameworkProperties,
+                sourceOperations);
+
+        deleteOperations = new DeleteOperations(frameworkProperties,
+                queryOperations,
+                sourceOperations,
+                opsSecurity,
+                opsMetacard,
+                opsCatStore);
+
+        // set up deleteRequest
+        MetacardImpl metacard = new MetacardImpl();
+        ArrayList<Metacard> metacardList = new ArrayList<>();
+        metacard.setId("Bob");
+        metacard.setTitle("Bob's Title");
+        metacardList.add(metacard);
+
+        metacard = new MetacardImpl();
+        metacard.setId("Bobbert");
+        metacard.setTitle("Bobbert's Title");
+        metacardList.add(metacard);
+
+        deleteRequest = new DeleteRequestImpl(metacard.getId());
+        deleteResponse = new DeleteResponseImpl(deleteRequest, new HashMap(), metacardList);
+
+        // set up blacklist
+        fanoutTagBlacklist = new ArrayList<>();
+        fanoutTagBlacklist.add("");
+
+        opsSecuritySupport = mock(OperationsSecuritySupport.class);
+
+        opsMetacardSupport = mock(OperationsMetacardSupport.class);
+
+        opsCatStoreSupport = mock(OperationsCatalogStoreSupport.class);
+
+        remoteDeleteOperations = new RemoteDeleteOperations(frameworkProperties,
+                opsMetacardSupport,
+                opsCatStoreSupport);
+
+    }
+
+    @Test
+    public void testIsNotCatalogStoreRequest() throws Exception {
+
+        when(opsCatStoreSupport.isCatalogStoreRequest(deleteRequest)).thenReturn(false);
+        remoteDeleteOperations.setOpsCatStoreSupport(opsCatStoreSupport);
+
+        DeleteResponse resultDeleteResponse = remoteDeleteOperations.performRemoteDelete(
+                deleteRequest,
+                deleteResponse);
+
+        assertThat("Assert return of identical deleteResponse",
```

Comment:
Probably want to compare the metacard-ids on the metacards. Generally we don't want to do `==` on objects.